assets/styles/components/_tooltip.scss

```diff
@@ -0,0 +1,110 @@
+:root {
+  --tooltip-bg-start: #9130A8;
+  --tooltip-bg-end: #{$ddpurple};
+  --tooltip-text-color: #ffffff;
+  --tooltip-link-color: rgba(255, 255, 255, 0.7);
+  --tooltip-transition-duration: 200ms;
+  --tooltip-box-shadow: 0 4px 15px rgba(99, 44, 166, 0.1), 0 2px 5px rgba(99, 44, 166, 0.08);
+}
+
+.tooltip-container {
+  position: relative;
+  display: inline-block;
```

Comment:
@StefonSimmons - This seems to force a space after the shortcode. For example, imagine a scenario where we want a period directly after the tooltip:

![image](https://github.com/user-attachments/assets/8f0f4c3a-2bac-479e-8042-6e1e2093a0f1)

I pushed up a change where I revert inline-block and keep anchorize to see how it looks on Staging. This combo works locally.

Reply:
Ok, when I revert inline block, it's back to not allowing any spaces. I'm going to put it back to inline because that is more important than being able to use this at the end of a sentence for now.